### PR TITLE
Fix issue #746

### PR DESCRIPTION
**Pull request by daliwei**
cc @sbrossie @pierre

### CODE REVIEW

**Comment by sbrossie** [comment]
👍

**Comment by daliwei** [comment]
i tried to add one but didn't see much relevant tests on the jax-rs module though.

**Comment by sbrossie** [comment]
@daliwei Integration tests for jaxrs can be found under [profile](https://github.com/killbill/killbill/tree/master/profiles/killbill/src/test/java/org/killbill/billing/jaxrs)

**Comment by daliwei** [comment]
Added tests in 3ecdeb1

File: profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestPaymentPluginProperties.java [hunk 1]
```diff
@@ -103,18 +103,18 @@ private void assertPluginProperties(final Iterable<org.killbill.billing.payment.
             for (org.killbill.billing.payment.api.PluginProperty input : properties) {
                 boolean found  = false;
                 for (org.killbill.billing.payment.api.PluginProperty expect : expectedProperties) {
-                    if (expect.getKey().equals(input.getKey()) && expect.getValue().equals(expect.getValue())) {
+                    if (expect.getKey().equals(input.getKey()) && expect.getValue().equals(input.getValue())) {
```
Comment (pierre) [inline profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestPaymentPluginProperties.java]: 🤦‍♂️

**Comment by pierre** [comment]
Thanks!

**Pull request merged**